lib/floki/raw_html.ex

```diff
@@ -88,11 +89,7 @@ defmodule Floki.RawHTML do
   end
 
   defp build_attrs({attr, <<value::binary>>}, attrs) do
-    if String.contains?(value, "\"") do
-      ~s(#{attrs} #{attr}='#{value}')
-    else
-      ~s(#{attrs} #{attr}="#{value}")
-    end
+    "#{attrs} #{attr}=\"#{html_escape_attribute_value(value)}\""
```

Comment:
Always wrap with `"` instead of alternate between the two. The original issue (#181) that introduces this `if` should also be covered by this fix.